src/containers/Dashboard.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Chart } from 'primereact/chart';
+import '@binaris/shift-babel-macro/macro';
+
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+
+import {
+  generateRemoteData,
+  setStorageData,
+  getStorageData,
+} from '../../backend/dashboard-server';
+import { Display } from '../constants/enums';
+
+export default class Dashboard extends React.Component {
```

Comment:
Can we please use functional components with hooks?
We generally prefer this form.